File: 2498-frog-jump-ii/2498-frog-jump-ii.cpp
class Solution {
public:
     int maxJump(vector<int>& A) {
        int res = A[1] - A[0], n = A.size();
        for (int i = 2; i < n; ++i)
            res = max(res, A[i] - A[i - 2]);
        return res;
    }
};

File: 2498-frog-jump-ii/2498-frog-jump-ii.java
class Solution {
    public int maxJump(int[] A) {
        int res = A[1] - A[0], n = A.length;
        for(int i = 2; i < n; ++i)
            res = Math.max(res, A[i] - A[i - 2]);
        return res;
    }
}